content/what-is-cross-origin-resource-sharing/index.md

```diff
@@ -0,0 +1,166 @@
+---
+title: "What is Cross Origin Resource Sharing (CORS)"
+description: "Cross-Origin Resource Sharing (CORS) is a mechanism that supports secure requests and data transfers from outside origins (domain, scheme, or port)."
+date: "2022-10-27"
+cover: "cors-cover.png"
+category: "featured"
+author: "SuperTokens team"
+---
+
+## Table Of Contents:
+  - [Introduction](#introduction)
+  - [CORS History](#cors-history)
+  - [CORS preflight request](#cors-preflight-request)
+  - [CORS actual request](#cors-actual-request)
+  - [Wildcards in CORS](#wildcards-in-cors)
+  - [CORS vulnerability](#cors-vulnerability)
+  - [Conclusion](#conclusion)
+
+
+## Introduction 
+Cross-Origin Resource Sharing (CORS) is a mechanism that supports secure requests and data transfers from outside origins (domain, scheme, or port).
+
+For example, `example.com` uses a text font that's hosted on `fonts.com`. When visiting `example.com`, the user's browser will make a request for the font from `fonts.com`. If `fonts.com` allows cross-origin resource sharing to `example.com`, then the browser will proceed with loading the font. Otherwise, the browser will cancel the request.
```

Comment:
We introduce the term cross-origin here without explaining what it means - this can be confusing

Reply:
@snowypines missed this?